website/docs/development/developing/spark/built_in_lineage.md

```diff
@@ -186,84 +184,4 @@ will enrich dataset with `customFacet`:
     },
     "schema": { }
 }]
-```
-        
-## Column Level Lineage
```

Comment:
May be a silly question: what happened to column level lineage within built-in approach?